RobotInterrogation/appsettings.json

```diff
@@ -420,364 +437,406 @@
             ]
           },
           {
             "type": "Human",
+            "fault": "",
             "traits": [
               "You have nothing to hide."
             ]
           },
           {
             "type": "Human",
+            "fault": "",
             "traits": [
               "You have nothing to hide."
             ]
           },
           {
             "type": "Human",
+            "fault": "",
             "traits": [
               "You have nothing to hide."
             ]
           },
           {
             "type": "Human",
+            "fault": "",
             "traits": [
               "You have nothing to hide."
             ]
           },
           {
             "type": "ViolentRobot",
+            "fault": "Synthesis",
             "traits": [
-              "Describe how 3 things seem unfair to you.",
-              "2 times, say what you think should have happened instead.",
+              "3 times, add an element of a nonliving thing to a living thing",
+              "3 times, explain how something is a cross between two living things",
               "Perform the penalty twice."
             ]
           },
           {
             "type": "ViolentRobot",
+            "fault": "World-building",
             "traits": [
-              "3 times, blame someone else for something.",
-              "3 times, blame yourself for something.",
+              "Mention the same imagined thing in response to 3 questions",
+              "Mention 3 imaginary locations.",
               "Perform the penalty twice."
             ]
           },
           {
             "type": "ViolentRobot",
+            "fault": "Simplicity",
             "traits": [
-              "Describe dealing with 3 different tragedies the same way.",
-              "Refer to the same friend or family member 4 times.",
+              "Give one-word answers to 3 questions, and don't elaborate until asked",
+              "Answer 2 questions non-verbally, and stay silent until prompted",
               "Perform the penalty twice."
             ]
           },
           {
             "type": "PatientRobot",
+            "fault": "Sensation",
             "traits": [
-              "You may not say the words \"think,\" \"thought\", \"feel\" or \"felt.\""
+              "You may not describe how anything smells, tastes, sounds, or feels -- only how it looks."
             ]
           },
           {
             "type": "PatientRobot",
+            "fault": "Flexibility",
             "traits": [
-              "You must describe emotions only using physical descriptions rather than names, e.g. \"crying\" instead of \"sad\" or \"sweaty palms\" instead of \"nervous.\""
+              "Choose one:",
+              "If you introduce something new to the conversation, it must be a living thing,",
+              "or",
+              "If you introduce something new to the conversation, it must be a non-living thing."
```

Comment:
Unsure on how to best format this.

Reply:
If it looks OK on the site, I'd say the way you've done it looks the clearest to me.
We _could_ go so far as adding markdown support for formatting the choice, but that might be overkill.

Reply:
Yeah that seems fine to me.
![image](https://user-images.githubusercontent.com/5936748/77958415-db169e00-7289-11ea-8714-c06551880612.png)